src/components/EnterpriseApp/index.jsx

```diff
@@ -113,6 +114,9 @@ class EnterpriseApp extends React.Component {
       );
     }
 
+    if (loading) {
```

Comment:
think it should be simple to test that by passing a loading:true to the `<EnterpriseApp>` component that the output contains whatever matches the loading component. you can add a test at https://github.com/edx/frontend-app-admin-portal/blob/363a412774d680d5ee6f48214deb4c8e3cb0b2b4/src/containers/EnterpriseApp/EnterpriseApp.test.jsx#L48